packages/channels/src/model/open-dialog-result.ts

```diff
@@ -16,25 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { inject, injectable } from 'inversify';
-
-import { CONTEXT_HEALTHS } from '@kubernetes-contexts/channels';
-import { RpcBrowser } from '@kubernetes-contexts/rpc';
-
-import { AbsStateObjectImpl, type StateObject } from './util/state-object.svelte';
-import type { ContextsHealthsInfo } from '@podman-desktop/kubernetes-dashboard-extension-api';
-
-// Define a state for the ContextsHealthsInfo
-@injectable()
-export class StateContextsHealthsInfo
-  extends AbsStateObjectImpl<ContextsHealthsInfo, void>
-  implements StateObject<ContextsHealthsInfo, void>
-{
-  constructor(@inject(RpcBrowser) rpcBrowser: RpcBrowser) {
-    super(rpcBrowser);
-  }
-
-  async init(): Promise<void> {
-    await this.initChannel(CONTEXT_HEALTHS);
+export interface OpenDialogResult {
+    // File paths as strings
+    files: string[] | undefined;
   }
```

Comment:
I think you will need to handle the case when the user closes the file browser without choosing any file, with a `cancel?; boolean` field here?